### PR TITLE
Binding for Math.cos,cosh,exp,expml,fround,imul,log,log10,log1p,log2

### PR DESCRIPTION
**Pull request by autodidaddict**
Bindings and tests for several math functions.

### CODE REVIEW

**Comment by autodidaddict** [comment]
I don't know why the build seems to have failed? All the new tests work locally.

**Comment by jonathan-s** [comment]
`No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.`

So it's an issue with the Travis in this case.

**Comment by autodidaddict** [comment]
Can we re-run that build?

File: tests/all/js_globals/Math.rs [hunk 1]
```diff
@@ -293,6 +294,117 @@ fn clz32() {
...
+                js::Math::cos(x)
+            }
+        "#)
+        .file("test.ts", r#"
+            import * as assert from "assert";
+            import * as wasm from "./out";
+
+            export function test() {
+                assert.equal(wasm.cos(0), 1);
+                // other assertions failing due to rounding errors
```
Comment (jonathan-s) [inline tests/all/js_globals/Math.rs]: If want to make more assertions what you could do is the following:
```
assert.equal(wasm.cos(1.34), Math.cos(1.34)))
```
The same goes for the other assertions where you have a very precise float. Doing the above could make it a bit more easy to read as well :)

File: tests/all/js_globals/Math.rs [hunk 2]
```diff
@@ -78,6 +78,7 @@ fn acosh() {
             export function test() {
                 assert.equal(wasm.acosh(1), 0);
                 assert.equal(wasm.acosh(2), Math.acosh(2));
+                assert.equal(wasm.acosh(2), Math.acosh(2));
```
Comment (jonathan-s) [inline tests/all/js_globals/Math.rs]: This seems to be a line added by mistake?

Comment (autodidaddict) [reply tests/all/js_globals/Math.rs]: I didn't even work on `acosh` .. weird.

**Comment by autodidaddict** [comment]
Updated in response to your comments.

**Comment by autodidaddict** [comment]
Next time I will definitely split the commits per function. Not sure if I can do that retroactively

**Pull request merged**